dhis-2/dhis-services/dhis-service-analytics/src/main/java/org/hisp/dhis/analytics/DataQueryParams.java

```diff
@@ -1983,25 +1992,26 @@ public boolean equals( Object object )
             {
                 return false;
             }
+            if ( filters == null )
+            {
+                return other.filters == null;
+            }
+            else
+                return filters.equals( other.filters );
         }
-        else if ( !dimensions.equals( other.dimensions ) )
-        {
-            return false;
-        }
-
-        if ( filters == null )
+        else
         {
-            if ( other.filters != null )
+            if ( !dimensions.equals( other.dimensions ) )
             {
                 return false;
             }
+            if ( filters == null )
+            {
+                return other.filters == null;
+            }
+            else
+                return filters.equals( other.filters );
```

Comment:
Add brackets { } even for one-liners per the code style.

Reply:
Done